### DISCUSSION TITLE
Any news on SHVDN 3.7+?

### DISCUSSION THREAD

**Discussion by Ethirix** [scripthookvdotnet/scripthookvdotnet#1326] [Q&A] [answered]
A while ago I asked in an issue about the Euphoria Helper classes not working, to which I was told that they are actually fixed, just after 3.6 was released.

Is there any news on when a possible SHVDN 3.7 or other is going to be released as this mod is stuck in limbo until then? 
I don't want to release a mod that people then have to compile SHVDN from Source to use.

**Comment by kagikn** [comment, marked as answer]
Yeah I answered like [3.7 would be released in 1 week](https://github.com/scripthookvdotnet/scripthookvdotnet/issues/1259#issuecomment-1620111731)... But you will have to wait, and this time I can't really guarantee the ETA. The problem is, new API designs would be such a mess if I release 3.7 right now. [There are more than 150 commits since July 4th this year](https://github.com/scripthookvdotnet/scripthookvdotnet/commits/main?since=2023-07-05&until=2023-10-24). Without reviewing, some poor ones would bother me (and maybe you) since we can't really do some breaking changes in the same major version once new APIs are shipped in stable releases. If I had pushed commits that only contain hotfixes or performance improvements but no new API additions since v3.6, I would release v3.7 immediately.

**Reply by Ethirix** [reply]
Understandable, I just wanted to know your stance considering what you said prior, thank you.
